src/commands/deploy/showCoreToolsWarning.ts

```diff
@@ -10,52 +10,52 @@ import { FuncVersion, funcVersionLink, tryParseFuncVersion } from "../../FuncVer
 import { localize } from "../../localize";
 import { getWorkspaceSetting, updateGlobalSetting } from "../../vsCodeConfig/settings";
 
-export async function showCoreToolsWarning(context: IActionContext, projectVersion: FuncVersion): Promise<void> {
+export async function showCoreToolsWarning(context: IActionContext, projectVersion: FuncVersion, projectName: string): Promise<void> {
```

Comment:
nit: Technically, this isn't the project (which refers to the local project) but the function app itself.

```suggestion
export async function showCoreToolsWarning(context: IActionContext, projectVersion: FuncVersion, siteName: string): Promise<void> {
```